web/src/message_list_tooltips.js

```diff
@@ -234,4 +235,34 @@ export function initialize() {
             instance.destroy();
         },
     });
+
+    message_list_tooltip(".message_edit_notice", {
+        trigger: "mouseenter",
+        delay: LONG_HOVER_DELAY,
+        popperOptions: {
+            modifiers: [
+                {
+                    name: "flip",
+                    options: {
+                        fallbackPlacements: "bottom",
+                    },
+                },
+            ],
+        },
+        onShow(instance) {
+            const $elem = $(instance.reference);
+            const edited_notice_str = $elem.attr("data-tippy-content");
+            const $markup = $("<span>").text(edited_notice_str);
+
+            if (page_params.realm_allow_edit_history) {
+                const hint_line_str = $t({defaultMessage: "Click to view edit history."});
+                $markup.append($("<br>"), $("<i>").text(hint_line_str));
+            }
+
+            instance.setContent($markup[0]);
```

Comment:
Is it possible to use a template for this?

Reply:
@amanagr I have updated the PR, could you please review the PR, thanks ;)